Moderation-Policy.md

```diff
@@ -1,154 +1,305 @@
-## Moderation policy
-
-If you are not a member of the Node.js GitHub Organization and wish to submit a moderation request, please see [Requesting Moderation](#requesting-moderation)
-
-* [Applicability](#applicability)
-* [Terms](#terms)
-* [Grounds for Moderation](#grounds-for-moderation)
-* [Requesting Moderation](#requesting-moderation)
-* [Consideration of Intent](#consideration-of-intent)
-* [Guidelines and Requirements](#guidelines-and-requirements)
-  * [Collaborator Posts](#collaborator-posts)
-  * [Non-Collaborator Posts](#non-collaborator-posts)
-* [Escalation to the TSC](#escalation-to-the-tsc)
-* [Privacy of the nodejs/moderation Repository](#privacy-of-the-nodejsmoderation-repository)
-* [TSC Delegation of Authority to a "Moderation Working Group"](tsc-delegation-of-authority-to-a-moderation-working-group)
-* [Modifications to this Policy](#modifications-to-this-policy)
-
-### Applicability
-
-By default, this policy applies to all repositories under the Node.js GitHub Organization and all Node.js Working Groups.
-
-Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
-* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
-* Includes provisions for clearly and openly documenting Moderation actions taken.
-
-If a particular repository can be considered to fall under the stewardship of multiple Working Groups (or Top Level Projects) that have adopted different Moderation Policies, they can choose to either:
-* Decide between themselves which Moderation Policy will be in effect for the repository in question, or
+# Moderation policy
+
+If you are not a member of the Node.js GitHub Organization and wish to submit a
+moderation request, please see [Requesting Moderation][]
+
+* [Applicability][]
```

Comment:
Won't these links break? 

I would suggest using [doctoc](https://github.com/thlorenz/doctoc) to automatically make a TOC here.